concourse/pipelines/hadoop-cluster-cleaner.yml

```
@@ -25,17 +25,17 @@ resources:
     branch: ((pxf-git-branch))
     uri: ((ud/pxf/common/git-remote))
 
-- name: slack-alert
+- name: gchat-alert
   type: slack-notification
   source:
-    url: ((ud/pxf/secrets/ud-pipeline-bot-ud-dev-webhook))
+    url: ((ud/pxf/secrets/ud-pipeline-bot-gp-ud-gchat-webhook))
```

```suggestion
    url: ((ud/pxf/secrets/gchat-gp-extended-federation-ci-webhook))
```
the "CI" webhook for a given GChat space